lib/Attribute.js

```diff
@@ -427,6 +427,11 @@ Attribute.prototype.toDynamo = function(val, noSet, model) {
     }
 
     if(type.dynamo === 'S') {
+      if (this.options.enum) {
```

Comment:
This looks to be in the `toDynamo` function. Do we think it would be beneficial to add it to the `fromDynamo` function as well? Not sure if it makes sense, what are your thoughts on this?

Reply:
@brandongoode can you take a look at this and let me know your thoughts? Do you think we should add this to `fromDynamo` as well? Should we throw an error in the `fromDynamo` if it's not correct in the enum, or?